File: python/animals/owca.py
from animals.zwierze import Zwierze


class Owca(Zwierze):

    def __str__(self):
        return self._species + "[" + str(self._x) + "][" + str(self._y) + "]"

    def __init__(self, x, y, world, sila=4):
        Zwierze.__init__(self, "owca", sila, 4, x, y, world)

    def rysowanie(self):
        return "O"

File: java/src/Zwierzeta/Owca.java
package Zwierzeta;
import Swiat.*;

public class Owca extends Zwierze{
    public Owca(int x, int y , Swiat swiat){
        super(Gatunek.OWCA, 4,4, x, y, swiat);
    }
    public Owca(int x, int y, int sila, Swiat swiat){
        super(Gatunek.OWCA, sila,4, x, y, swiat);
    }

    @Override
    public String rysowanie(){return "O";}
}
